crates/devolutions-pedm/openapi/dotnet-client/config.json

```diff
@@ -2,7 +2,7 @@
   "packageAuthors": "Devolutions Inc.",
   "packageName": "Devolutions.Pedm.Client",
   "packageTitle": "Devolutions PEDM REST API Client",
-  "packageVersion": "2024.10.25",
+  "packageVersion": "2024.3.6.0",
```

Comment:
For the Devolutions.Pedm.Client library, I suggest we keep following the date format that we follow for all the other libraries. 2024.3.6.0 (and so on) would only be for user-facing applications that we actually ship.

Reply:
Yes you are right, I missed this detail. Will correct that.

Reply:
Thank you! Feel free to rebase-merge afterwards! :slightly_smiling_face:

Reply:
@CBenoit Please check the last commit. I updated the generate-clients script to automatically bump the version to today's date.

Reply:
Good idea! Do you think you could update all the config.json files instead of only the first?

Reply:
You mean, also the `devolutions-pedm-client-http` config?

Is it what we want? As far as I can tell, the version is useless here because it's only consumed as a reference by it's parent crate...

Reply:
My bad, you are right, all the other packages are following a different convention.